modules/apps/commerce/commerce-product-definitions-web/src/main/java/com/liferay/commerce/product/definitions/web/internal/frontend/data/set/provider/CommerceProductDefinitionSpecificationFDSDataProvider.java

```diff
@@ -89,6 +92,29 @@ public int getItemsCount(
 			getCPDefinitionSpecificationOptionValuesCount(cpDefinitionId, null);
 	}
 
+	private String _getCPDefinitionSpecificationOptionValue(
+		CPDefinitionSpecificationOptionValue
+			cpDefinitionSpecificationOptionValue,
+		String languageId) {
+
+		if (cpDefinitionSpecificationOptionValue.
+				getAvailableLanguageIds().length == 1) {
```

Comment:
Avoid chaining, use a variable for `cpDefinitionSpecificationOptionValue.getAvailableLanguageIds()`